docs/application/vstools/tools/project-wizard.md

```diff
@@ -1,53 +1,53 @@
 ﻿# Project Wizard
 
-The Project Wizard in Visual Studio Tools for Tizen is used to create a Tizen .NET application project.
+You can use Project Wizard to create Tizen .NET application projects. In Project Wizard, you can select a template and a profile to create the project. Aditionally, you can set the basic project properties, such as the project name, package name, location, and so on.
 
-In the Project Wizard, you can select a template and profile to create the project. In addition, you can set basic project properties, such as the project name, package name, and location.
+The Project Wizard provides the following project templates using which you can start a Tizen application project:
 
-The Project Wizard provides 4 project templates with which you can start a .NET application project:
+Template | Description | Supported Versions
```

Comment:
Question about this table itself.

It looks like in this table, most "Description" sentences are simply re-written version of the template name. The "Supported Versions" and the Template list itself are distinguishable information. So, should we use the table form, or add "Description" section? I guess a simple list of "project template" would work the same. How about changing it to a list with a sentence like "Tizen OpenTK App is only supported by Tizen 5.0"? What do you think?

Reply:
I agree with your point. However, I had earlier used a simple list but that increases the page length. In my opinion, having a table format allowed all the information to be in one block. I will brainstorm further on how we can represent this information better. Request you to also please  guide me if there could be another way.

Reply:
I'll also brainstorm about it.

Reply:
Thank you. I will make the changes discussed so far and commit.